packages/sd-jwt/src/verify.ts

```diff
@@ -84,10 +85,91 @@ async function verifyJWT(
   });
 }
 
+/**
+ * Parse an encoded disclosure token into a triple [salt, claim name, claim value].
+ *
+ * @example
+ * // returns ["rSLuznhiLPBDRZE1CZ88KQ", "sub", "john_doe_42"]
+ * parseDisclosureToken("WyJyU0x1em5oaUxQQkRSWkUxQ1o4OEtRIiwgInN1YiIsICJqb2huX2RvZV80MiJd")
+ *
+ * @param token A base64 string representing the disclosure
+ * @returns The triple representing the parsed disclosure
+ * @throws An error in case the token is failed to decode and parse
+ */
+function parseDisclosureToken(token: string): Disclosure {
+  // token is a base64 string
+  const decoded = atob(token);
```

Comment:
`atob` is [deprecated](https://nodejs.org/api/buffer.html#bufferatobdata).
Use `Buffer.from(data, "base64")` instead!

Reply:
The point is to be consistent in a non-Node.js environment. 

I'm not quite an expert, I just investigated a bit. It seems the problem with `atob` is it's not consistent for every char in the `utf-8` space (https://developer.mozilla.org/en-US/docs/Glossary/Base64#the_unicode_problem). That is Unicode char that are represented by more than a byte, like emojis.

However, maybe we are safe given the specificity of our use case. 

I think we can live with `atob` for now, and keep investigating. When we'll find a more solid solution, I think the rework would only cost a method replacement.